src/common/crypto/platform/openssl/crypto.cpp

```diff
@@ -478,21 +478,61 @@ expected::ExpectedString ExtractPublicKey(const Args &args) {
 	if (ret != OPENSSL_SUCCESS) {
 		return expected::unexpected(MakeError(
 			SetupError,
-			"Failed to extract the public key from: (" + args.private_key_path
+			"Failed to extract the public key from bio: (" + args.private_key_path
```

Comment:
To be consistent with the message below, shouldn't this be "... from _the_ bio _key_:" ? If it is intended to be different, we should explicitly say something else.

Reply:
All open to good suggestions here

Reply:
I changed them all to `from private key `. 

Then the `GetOpenSSLError` will hopefully return the useful parts.